CODEOWNERS

```diff
@@ -3,6 +3,7 @@
 /elements/pl-order-blocks @SethPoulsen
 /elements/pl-rich-text-editor @jonatanschroeder
 /elements/pl-dataframe @coatless @eliotwrobson
+/elements/pl-card @tdy @coatless
```

Comment:
Let's keep this list alphabetized (I realize we didn't do that in the dataframe PR - oops!)

Reply:
Done eecf0e04cb662c6fb9992aaaf80169db997ec52c